src/main/java/org/folio/rest/impl/DataExportHelper.java

```diff
@@ -28,15 +28,19 @@
 
   private Future<String> getDownloadFileName(String jobExecutionId, String exportFileId, String tenantId) {
     return jobExecutionService.getById(jobExecutionId, tenantId)
-      .map(job -> job.map(jb -> jb.getExportedFiles()
+      .map(job -> job.getExportedFiles()
         .stream()
         .filter(expFile -> expFile.getFileId()
           .equals(exportFileId))
-        .findFirst())
+        .findFirst()
         .orElseThrow(() -> new ServiceException(HttpStatus.HTTP_NOT_FOUND, String.format("Job with id: %s not found", jobExecutionId))))
-      .map(exportedFile -> exportedFile.map(ExportedFile::getFileName)
-          .orElseThrow(() -> new ServiceException(HttpStatus.HTTP_NOT_FOUND, String.format("Export File with id: %s not found: ", exportFileId))));
-
+      .compose(exportedFile -> {
+        if (StringUtils.isEmpty(exportedFile.getFileName())) {
```

Comment:
fileName is a mandatory field in schema, this check is not required

Reply:
removed this check, thank you